README.md

```diff
@@ -5,14 +5,25 @@ This is the codebase for the canonical.design static website
 ## Architecture overview
 
 This site is currently under-development.
+
 ## Development
 
 The simplest way to run the site is:
 
-- Create a python virtual environment
-- install requirements
-- install node modules
-- run `yarn build`
-- run `flask run` in webapp folder
+- Create a python virtual environment (make sure you are using python3.8)
```

Comment:
I believe 3.10 is also fine

Reply:
Have you tested it? I am not sure it is compatible with flask-base. I tested with 3.9 and it did not work